src/main/java/com/khadri/jdbc/prepare/data/student/dao/JdbcConnectionUtil.java

```diff
@@ -0,0 +1,30 @@
+package com.khadri.jdbc.prepare.data.student.dao;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+
+public class JdbcConnectionUtil {
+
+	private static Connection con;
+
+	public static Connection getConnection() {
```

Comment:
implement the logic to read from db.properties and pass to below DriverManager object creation